extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -505,9 +505,8 @@ export class AmpA4A extends AMP.BaseElement {
    * signing service(s) is to be used.
    * @return {!Array<string>} A list of signing services.
    */
-  getSigningServiceNames() {
-    // TODO(levitzky) Add dev key name once it goes live.
-    return getMode().localDev ? ['google'] : ['google'];
+  getSigningServiceNames_() {
+    throw new Error('getSigningServiceNames not implemented!');
```

Comment:
That's a fair point. I'll hold off on this for a bit until I have more information.